nesta/core/scripts/docker_recipes/py36_sklearn_amzn1_image/Dockerfile

```diff
@@ -1,25 +1,55 @@
 FROM amazonlinux:1
 
+RUN du -cxh --threshold=5M --max-depth=3 /
```

Comment:
I understand what the command does but not why it is here/what it does in the context of a Dockerfile

Reply:
(and for lines 42,43)